apps/authentication-portal/src/main/webapp/requested-claims.jsp

```diff
@@ -69,6 +69,12 @@
                     <%=AuthenticationEndpointUtil.i18n(resourceBundle, "provide.mandatory.details")%>
                 </h3>
 
+                <% if (request.getParameter("postAuthMissingClaimsError") != null) { %>
+                    <div class="ui visible negative message" id="error-msg" data-testid="page-error-message">
+                        <%= Encode.forHtmlContent(request.getParameter("postAuthMissingClaimsError")) %>
```

Comment:
Can you check whether calling 
 works?

So that we can support localization as well. 

The expectation is if someone wants localization support, they can add the localization key as the `RegExValidationError` in claims property.

Reply:
fixed in 9ea547d